MsixCore/msixmgr/CommandLineInterface.cpp

```diff
@@ -175,6 +175,19 @@ std::map<std::wstring, Options, CaseInsensitiveLess> CommandLineInterface::s_opt
                     return S_OK;
                 }),
             },
+            {
+                L"-output",
+                Option(true, IDS_STRING_HELP_OPTION_OUTPUT,

```

Comment:
>IDS_STRING_HELP_OPTION_OUTPUT [](http://example.com/codeflow?start=29&length=29)

Since this option is unpack only I'd rename these to fit the unpack specific options elsewhere, so IDS_STRING_HELP_OPTION_UNPACK_OUTPUT

Reply:
fixed